Cabal/src/Distribution/Simple.hs

```diff
@@ -157,7 +157,9 @@ defaultMainWithHooksNoReadArgs hooks pkg_descr =
 defaultMainHelper :: UserHooks -> Args -> IO ()
 defaultMainHelper hooks args = topHandler $ do
   args' <- expandResponse args
-  case commandsRun (globalCommand commands) commands args' of
+  command <- commandsRun (globalCommand commands) commands args'
+  case command of
+    CommandDelegate -> pure ()
```

Comment:
Mhhh could this be abstracted with the one in `Cabal/src/Distribution/Make.hs` above?

Reply:
I'm not sure this PR-contributor combo is the appropriate one for such a refactoring. :)